### ISSUE TITLE
Training with the lightstereo model result in 'nan' loss in the later stages

### ISSUE THREAD

**Issue by leoShen917** [XiandaGuo/OpenStereo#175]
Great work! I use the lightstereo model and train it on the sceneFlow dataset using the official cfgs file. But after training for one or two epochs, the training loss becomes nan. May I ask what caused this? Looking forward to your reply @XiandaGuo

**Comment by valeriopaolicelli** [comment]
Same situation with lightstereo_m

**Comment by leoShen917** [comment]
Solved. The attention module is not well designed, which leads to numerical explosion of the forward process at the end of training. Attention can be followed by a tanh or softmax activation function.
